### ISSUE TITLE
Update work item flow failing because of empty Epic `state`

### ISSUE THREAD

**Issue by maririos** [Azure/azure-sdk-tools#9123]
See failures in runs: https://make.powerautomate.com/environments/ed2ffefd-774d-40dd-ab23-7fff01aeec9f/flows/0ded20bb-b79f-939b-4fb9-82cdb404de0a/runs?filter=failed

Happens only for the `Epic` work item and the filed `State`

**Comment by maririos** [comment]
Looks like the Epic itself is created with empty values for `ServiceTree Id` and `Service lifecycle`. I verified that for all services we do have this information, and that other `Service` properties are correctly set.
I did find that the value that was use to update the `State` of an `Epic` was coming from the `TriageWorkItem` which is in memory. We already had a property called `ServiceState` so I changed it to that so it is clear what the content is.
I don't think this will fix the issue though. I am going to deploy it and keep monitoring
